draft-yasskin-httpbis-origin-signed-exchanges-impl.md

```diff
@@ -803,10 +821,21 @@ able to make even one unauthorized signature.
 Certificates with this extension MUST be revoked if an unauthorized entity is
 able to make even one unauthorized signature.
 
+Starting 2019-05-01, certificates with this extension MUST have a Validity Period no greater than 90
```

Comment:
Should we align to draft-yasskin-http-origin-signed-responses.md here to rely on "The above requirements on CAs..." paragraph?